### DISCUSSION TITLE
Collapsible `Panel` as first field won't collapse

### DISCUSSION THREAD

**Discussion by ekateiva** [laravel/nova-issues#6646] [Ideas & Feature Requests]
- Laravel Version: 11.30.0
- Nova Version: v4.35.4
- PHP Version: 8.3.9
- Database Driver & Version: MySQL
- Operating System and Version: macOS 14.4
- Browser type and version: Chrome 130.0.6723.92 
- Reproduction Repository: N/a

### Description:

When you define a `Panel::make(...)->collapsable()` as the first `fields()` element, it is not collapsable and the arrow is not even seen.

<img width="1002" alt="image" src="https://github.com/user-attachments/assets/014baef0-66b0-4b50-b483-3aa1a8802a91">

### Detailed steps to reproduce the issue on a fresh Nova installation:

```
public function fields(NovaRequest $request)
{
        return [
            Panel::make('First Panel', [
                // Fields
            ])
            ->collapsable(),
            // ...
            // Other fields
        ];
}
```

### Related issue
https://github.com/laravel/nova-issues/issues/4749

**Comment by crynobone** [comment]
After much consideration, we are not going to change the behavior here. You can create an empty panel to cover the use case. In future versions, we will add a deprecation warning if you attempt to use it as the first panel and throw an exception in Nova 6.
